### PR TITLE
Migrate sap overview e2e

### PR DESCRIPTION
**Pull request by arbulu89**
Import SAP System overview page E2E tests, coming from the legacy trento project.

- Test view layout
- Test tagging filtering
- Test the health changes on events
- Test unknown sap instances like the diagnostic service (DAA) are not displayed

### CODE REVIEW

File: test/e2e/cypress/fixtures/sap-systems-overview/sap_systems_overview.feature [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    Background:
+        Given an healthy SAP deployment consisting of 27 hosts, 1 hana cluster, 1 netweaver cluster
+        Given 3 Application SAP Systems with sids 'NWP', 'NWQ', 'NWD'
+        Given 1 HANA database for each SAP System 'HDP', 'HDQ', 'HDD'
+        And a Trento installation on this Cluster
+
+    Scenario: Registered SAP Systems should be available in the overview
+        When I navigate to the SAP Systems overview page
+        Then the discovered SID ar the expected ones
```
Comment (nelsonkopliku) [inline test/e2e/cypress/fixtures/sap-systems-overview/sap_systems_overview.feature:12]: typo: 
`Then the discovered SID ar the expected ones`
`Then the discovered SID are the expected ones`

File: test/e2e/cypress/fixtures/sap-systems-overview/sap_systems_overview.feature [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        And I tag the SAP System with SID 'NWP' with 'env1'
+        And I tag the SAP System with SID 'NWQ' with 'env2'
+        And I tag the SAP System with SID 'NWD' with 'env3'
+        When I filter by tag
+        Then I should see only the SAP Systems with the tag I filtered by
+
+    Scenario: System health state is changed upon new SAP system events
+        Given I navigate to the SAP Systems overview page
+        When a new SAP system event for the first SAP system with the 1st instance with a GRAY status is received
+        And the page is refreshed
```
Comment (nelsonkopliku) [inline test/e2e/cypress/fixtures/sap-systems-overview/sap_systems_overview.feature:45]: Do we need `page refreshes`?

**Review by dottorblaster — COMMENTED** [review]
LGTM

**Review by nelsonkopliku — COMMENTED** [review]
I personally tend to prefer more specific selectors by adding some extra css classes or ids,  rather than traversing the DOM in this way'
```js
cy.get('td')
    .contains(sid)
    .parent('td')
    .parent('tr')
    .within(() => {
...
```

There's a discussion going on here on @rtorrero 's PR https://github.com/trento-project/web/pull/520#discussion_r870440310
Maybe that's just my perception.

Now, I know you ported what was there in legacy trento, and I am not pushing to do more work if not necessary :sweat_smile: 
Just sharing my thoughts.

File: test/e2e/cypress/fixtures/sap-systems-overview/sap_systems_overview.feature (hunk 1 above)
Comment (arbulu89) [reply test/e2e/cypress/fixtures/sap-systems-overview/sap_systems_overview.feature:12]: Damn!!
Time to add a syntax checker. 
I will create a tech-debt ticket in our backlog

File: test/e2e/cypress/fixtures/sap-systems-overview/sap_systems_overview.feature (hunk 2 above)
Comment (arbulu89) [reply test/e2e/cypress/fixtures/sap-systems-overview/sap_systems_overview.feature:45]: Nope. Leftover...

**Comment by arbulu89** [comment]
@nelsonkopliku I will create a tech-debt ticket in our backlog to address this if you don't mind.
I know, we shouldn't create more tech-debt while fixing tech-debt, but I find easier to do an "as it is" migration first, and apply improvements later.

**Comment by nelsonkopliku** [comment]
> @nelsonkopliku I will create a tech-debt ticket in our backlog to address this if you don't mind. I know, we shouldn't create more tech-debt while fixing tech-debt, but I find easier to do an "as it is" migration first, and apply improvements later.
>
Fine with me :smile:

**Pull request merged**